util/run-gnu-test.sh

```diff
@@ -49,6 +60,9 @@ if test "$1" != "run-root"; then
     # run the regular tests
     if test $# -ge 1; then
         timeout -sKILL 4h make -j "$(nproc)" check TESTS="$SPECIFIC_TESTS" SUBDIRS=. RUN_EXPENSIVE_TESTS=yes RUN_VERY_EXPENSIVE_TESTS=yes VERBOSE=no gl_public_submodule_commit="" srcdir="${path_GNU}" || : # Kill after 4 hours in case something gets stuck in make
+        echo "Pro tips:"
+        echo "add 'set -v' in $SPECIFIC_TESTS"
+        echo "add 'echo \$fail' in various places to verify when it starts to fail"
```

Comment:
I think this output is just noise because usually you don't need these tips, especially not, if the tests pass. Instead I would move them to `CONTRIBUTING.md`.

Reply:
ok, i will remove this